### PR TITLE
refine and align with pytorch

### PR DESCRIPTION
**Pull request by hjchen2**
- [x] conv1d/2d/3d和deconv1d/2d/3d kernel，stride和dilation参数支持 int 传参
- [x] 提供Tensor.zero_()接口
- [x] Tensor.norm 参数命名
- [x] flow.max和flow.min用法 
- [x] flow.nn.functional.dropout，支持inplace

### CODE REVIEW

File: python/oneflow/framework/tensor.py [hunk 1]
```diff
@@ -1176,6 +1180,7 @@ def RegisterMethods():
     Tensor.prod = _prod
     Tensor.sin = _sin
     Tensor.sin_ = _sin_inplace
+    Tensor.zero_ = _zero_
```
Comment (MARD1NO) [inline python/oneflow/framework/tensor.py:1183]: 我们还有个地方是flow.zero_但是接口是flow.zeros_，也顺便改下吧

上下文在：https://github.com/Oneflow-Inc/oneflow/issues/7573

File: oneflow/core/functional/functional_api.yaml [hunk 2]
```diff
@@ -228,6 +228,20 @@
   signature: "Tensor (Tensor dz, Tensor x, Tensor y) => XlogyYGrad"
   bind_python: False
 
+- name: "max"
+  signature: [
+    "TensorTuple (Tensor input, Int32 dim=None, Bool keepdim=False) => Max",
+    "Tensor (Tensor input, Tensor other) => Max"
```
[the remark this answers is not in the document]
Comment (hjchen2) [reply oneflow/core/functional/functional_api.yaml:234]: "TensorTuple (Tensor input, Int32 dim=None, Bool keepdim=False) => Max", 这个就支持你说的那种，输出如果是TensorTuple，而且size是1的话，就自动会取第0个Tensor作为输出

[the remark this answers is not in the document]
Comment (Flowingsun007) [reply oneflow/core/functional/functional_api.yaml:234]: 哦哦好的，忽略

**Review by MARD1NO — APPROVED** [review]

**Review by Flowingsun007 — APPROVED** [review]

**Review by wyg1997 — APPROVED** [review]

File: python/oneflow/framework/tensor.py (hunk 1 above)
Comment (hjchen2) [reply python/oneflow/framework/tensor.py:1188]: done，见commit https://github.com/Oneflow-Inc/oneflow/pull/7593/commits/204dd3fa31180e0cca6546be547e1d833be5212d
